packages/dso-toolkit/src/components/card/card.scss

```diff
@@ -12,32 +12,34 @@
 }
 
 dso-card {
-  &[is-selectable],
-  &[has-image] {
-    @extend %card-grid;
-  }
-
-  a[slot="heading"] {
+  .dso-card-heading a {
     text-decoration: none;
 
     dso-icon:last-child {
       align-self: flex-start;
     }
-  }
 
-  &:hover a[slot="heading"] {
-    color: card-variables.$heading-anchor-hover-color;
-    text-decoration: underline;
+    &:hover {
+      color: card-variables.$heading-anchor-hover-color;
+      text-decoration: underline;
+    }
   }
 }
 
 .dso-card,
 dso-card {
   @include card-mixins.base();
 
-  &.dso-is-selectable,
-  &.dso-has-image {
-    @extend %card-grid;
+  &.dso-is-selectable {
```

Comment:
Is het nodig dat naast `.dso-card` ook op `dso-card` wordt toegepast?

Reply:
Pak jij deze, @hansgrimm ?

Reply:
> Is het nodig dat naast `.dso-card` ook op `dso-card` wordt toegepast?

ja. als je die weghaalt dondert de core implementatie in mekaar.

Reply:
Maar `dso-card.dso-is-selectable` bestaat niet, hetzelfde geldt voor de `:has()` selector op regel 44

Reply:
weer dat brilletje... `.dso-card.dso-is-selectable` bestaat wel, maar zonder puntje idd niet.

Reply:
Maar, als ik `dso-card` weghaal, dan dondert zoals eerder opgemerkt. binnen die hele uitgebreide combined selector staan zaken die niet nodig zijn voor beide elementen, maar ook zaken die ze wel allebei nodig hebben. het enige dat ik kan doen hier, is de hele boel uit elkaar pluizen --> meer regels css.

Reply:
Bijvoorbeeld, dit zit er ook in, het ene voor de html/css, de andere voor core:
<img width="451" alt="Scherm­afbeelding 2024-10-10 om 21 52 05" src="https://github.com/user-attachments/assets/6ef577b0-1ae3-4e7a-82a2-800e03c55899">
Ik wil de boel best uit elkaar trekken, maar zo is het ooit gebouwd.

Reply:
@tfrijsewijk toch maar uit elkaar getrokken. is schoner zo, en duidelijker voor de generatie na ons ;)

Reply:
Klinkt goed! Dank